src/features/changeTweetActions.ts

```diff
@@ -16,42 +17,116 @@ export const changeTweetActionsStyling = makeBTDModule(({settings, jq, TD}) => {
-      usernamesAllowlist.length === 0
-    ) {
-      jq(document).trigger('uiShowFavoriteFromOptions', {
-        tweet: usefulChirp,
+      usefulChirp.favorite({
+        element: jq(e.target.closest('.stream-item')!),
+        statusKey: usefulChirp.id,
+        column: chirp.extra.columnKey,
       });
-      return;
-    }
+    });
+  }
+
+  if (settings.separateTweetAndQuoteTweetActions) {
+    modifyMustacheTemplate(TD, 'status/tweet_single_actions.mustache', (string) => {
+      const retweetOnlyItem = `<li class="tweet-action-item pull-left margin-r--10 {{#isProtected}}is-protected-action{{/isProtected}}">
+  <a class="tweet-action {{#isProtected}}no-pointer-events{{/isProtected}}" href="#"
+    data-btd-action="retweet-only" rel="retweet-only" title="Retweet only">
+    <i class="js-icon-retweet icon icon-retweet{{#isRetweeted}}-filled{{/isRetweeted}} icon-retweet-toggle txt-center {{#withPrettyEngagements}}pull-left{{/withPrettyEngagements}}"></i>
+    {{#withPrettyEngagements}}
+    <span class="pull-right icon-retweet-toggle margin-l--3 margin-t--1 txt-size--12 js-retweet-count retweet-count">
+      {{#prettyRetweetCount}}{{prettyRetweetCount}}{{/prettyRetweetCount}}
+    </span>
+    {{/withPrettyEngagements}}
+    <span class="is-vishidden">{{_i}}Retweet{{/i}}</span>
+  </a>
+</li>`;
+
+      const marker = '{{#getMainUser}}';
+      return string
+        .replace(
+          'icon-retweet{{#isRetweeted}}-filled{{/isRetweeted}} icon-retweet-toggle txt-center {{#withPrettyEngagements}}pull-left{{/withPrettyEngagements}}',
+          'icon-retweet txt-center' // icon-quote?
+        )
+        .replace(
+          '{{#withPrettyEngagements}} <span class="pull-right icon-retweet-toggle margin-l--3 margin-t--1 txt-size--12 js-retweet-count retweet-count">{{#prettyRetweetCount}}{{prettyRetweetCount}}{{/prettyRetweetCount}}</span> {{/withPrettyEngagements}}',
+          ''
+        )
+        .replace(marker, `${marker}${retweetOnlyItem}`);
+    });
+
+    modifyMustacheTemplate(TD, 'status/tweet_detail_actions.mustache', (string) => {
+      const retweetOnlyItem = `<li class="tweet-detail-action-item {{#isProtected}}is-protected-action{{/isProtected}}">
+  <a class="tweet-detail-action {{#isProtected}}no-pointer-events{{/isProtected}}" href="#"
+    data-btd-action="retweet-only" rel="retweet-only" title="Retweet only">
+    <i class="js-icon-retweet icon icon-retweet{{#isRetweeted}}-filled{{/isRetweeted}} icon-retweet-toggle"></i>
+    <span class="is-vishidden">{{_i}}Retweet{{/i}}</span>
+  </a>
+</li>`;
 
-    usefulChirp.favorite({
-      element: jq(e.target.closest('.stream-item')!),
-      statusKey: usefulChirp.id,
-      column: chirp.extra.columnKey,
+      const marker = '{{#getMainUser}}';
+      return string
+        .replace(
+          'icon-retweet{{#isRetweeted}}-filled{{/isRetweeted}} icon-retweet-toggle',
+          'icon-retweet' // icon-quote?
```

Comment:
`icon-quote` doesn't exist but we can easily make it ourselves.

```css
.icon-quote::before {
    content: "\F113";
}
```

Reply:
Neat, I didn't know there was already an icon for that!